### PR TITLE
Rename solution and projects

### PR DESCRIPTION
**Pull request by sutser**

### CODE REVIEW

File: CompositionRoot/CompositionRoot.cs [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ﻿using Autofac;
 using Autofac.Core;
 
-namespace Messerli.CompositionRootBuilder
+namespace Messerli.CompositionRoot
 {
-    public sealed class CompositionRootBuilder
+    public sealed class CompositionRoot
```
Comment (dngroth) [inline CompositionRoot/CompositionRoot.cs]: Class should still be called `CompositionRootBuilder`

File: CompositionRoot/CompositionRoot.csproj [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <AssemblyName>Messerli.CompositionRoot</AssemblyName>
+        <RootNamespace>Messerli.CompositionRoot</RootNamespace>
+    </PropertyGroup>
+    <PropertyGroup>
```
Comment (dngroth) [inline CompositionRoot/CompositionRoot.csproj]: Add nuget specs

Comment (sutser) [reply CompositionRoot/CompositionRoot.csproj]: Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/2/commits/baf96cbed727b79daafca4a394894fe55c928ba2
Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/2/commits/07c04859232b51fd2e61bc7d4d84b8c726924835

File: CompositionRoot/CompositionRoot.cs (hunk 1 above)
Comment (sutser) [reply CompositionRoot/CompositionRoot.cs]: Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/2/commits/d331ba69cf6b9235fcf22a4405d6ebaebe077d10

File: CompositionRoot/CompositionRoot.csproj [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <TargetFramework>netstandard2.1</TargetFramework>
+        <LangVersion>8.0</LangVersion>
+        <Nullable>enable</Nullable>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    </PropertyGroup>
+    <PropertyGroup>
+        <PackageId>Messerli.CompositionRoot</PackageId>
+        <Description>Class library for composition root to compose modules.</Description>
```
Comment (dngroth) [inline CompositionRoot/CompositionRoot.csproj]:
```suggestion
        <Description>Class library for composing composition root from modules.</Description>
```

**Pull request merged**